### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -178,10 +169,10 @@ for instructions).
 ## Developing Vega-Lite and Vega-Util
 
 Vega-Lite depends on [vega-util](https://github.com/vega/vega-util).
-If you plan to make changes to datalib and test Vega-Lite without publishing / copying compiled vega-util all the time, use [`npm link`](http://justjs.com/posts/npm-link-developing-your-own-npm-modules-without-tears) command.
+If you plan to make changes to the uils and test Vega-Lite without publishing / copying compiled vega-util all the time, use [`npm link`](http://justjs.com/posts/npm-link-developing-your-own-npm-modules-without-tears) command.
```
Comment (ayush29feb) [inline CONTRIBUTING.md]: is it suppose to be utils?

**Comment by domoritz** [comment]
What else? It's just an example. Maybe we should just delete the whole
block because it's not that relevant anymore.b

File: CONTRIBUTING.md (hunk 1 above)
Comment (domoritz) [inline CONTRIBUTING.md]: What else? It's just an example. Maybe we should just delete the whole
block because it's not that relevant anymore.

Comment (ayush29feb) [inline CONTRIBUTING.md]: Oh I mean it looks like a typo, you added uils

Comment (domoritz) [inline CONTRIBUTING.md]: Good catch. Fixed in master.

**Pull request merged**